<launch>
  <!-- set to value="gdbserver localhost:10000" for remote debugging -->
  <arg name="launch_prefix" default="" />

  <arg name="actual_image_topic" default="/camera/color/image" />
  <arg name="actual_info_topic" default="/camera/color/info" />
  <arg name="queue_size" default="1" />

  <node pkg="apriltag_ros" type="apriltag_ros_continuous_node" name="apriltag_ros_continuous_node" clear_params="true" output="screen" launch-prefix="$(arg launch_prefix)">
    <remap from="image_rect" to="$(arg actual_image_topic)" />
    <remap from="/camera/color/camera_info" to="$(arg actual_info_topic)" />

    <param name="parent_frame" type="string" value="camera_color_frame" />

    <param name="publish_tag_detections_image" type="bool" value="true" />
    <param name="queue_size" type="int" value="$(arg queue_size)" />

    <rosparam command="load" file="$(find apriltag_ros)/config/settings.yaml"/>
    <rosparam command="load" file="$(find apriltag_ros)/config/tags.yaml"/>
  </node>

  <node pkg="apriltag_ros" type="drone_tf_publisher.py" name="drone_tf_publisher" output="screen">
    <!-- 可以在这里添加节点参数 -->
  </node>


</launch>
